widgets/FileCopyToGCSAction-action.json

```diff
@@ -0,0 +1,105 @@
+{
+  "metadata": {
+    "spec-version": "1.0"
+  },
+  "display-name" : "Copy File to GCS",
+  "configuration-groups": [
+    {
+      "label": "Properties",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "Source Path",
+          "name": "sourcePath"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Project ID",
+          "name": "project",
+          "widget-attributes" : {
+            "default": "auto-detect"
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Destination Bucket",
+          "name": "destPath"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Wildcard",
+          "name": "fileRegex"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Number of parallel tasks",
```

Comment:
Move this to advanced section and display it only if expanded.

Reply:
I have moved it to advanced section. Not sure how to "display it only if expanded". Is there a widget configuration to do that?